server/src/main/kotlin/org/javacs/kt/SourceFiles.kt

```diff
@@ -106,7 +106,13 @@ class SourceFiles(private val sp: SourcePath) {
     private fun readFromDisk(file: Path): SourceVersion? {
         if (!Files.exists(file)) return null
 
-        val content = Files.readAllLines(file).joinToString("\n")
+        var content = ""
+
+        try {
+            content = Files.readAllLines(file).joinToString("\n")
+        } catch(exception:Exception) {
+            LOG.debug("Exception while parsing source file : ${file.toFile().absolutePath}")
```

Comment:
`LOG.warn` seems more appropriate to me, since this exception could provide valuable insights even when the language server is running at the default logging level.

Reply:
```suggestion
            LOG.warn("Exception while parsing source file : ${file.toFile().absolutePath}")
```

Reply:
I agree 👍